source/fundamentals/connection/tls.txt

```diff
@@ -77,6 +77,26 @@ in the following ways:
    see the :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
    section in the Server manual.
 
+.. note:: Workaround for an "unsafe legacy renegotiation disabled" error
```

Comment:
Q: is 'unsafe legacy renegotation disabled' the exact text in the error? is it capitalized there?

Reply:
Yes, this is how the error is written in the ticket (uncapitalized) and online in forums.